### PR TITLE
section2 complete with mobile responsiveness and UI is according to t…

### PR DESCRIPTION
**Pull request by RajGM**
…he design

Signed-off-by: RajGM <rajgmsocial19@gmail.com>

Conflicts resolved

### CODE REVIEW

**Comment by Tess99854** [comment]
@RajGM please rebase and fix the conflicts

File: pages/index.tsx [hunk 1]
```diff
@@ -31,7 +36,11 @@ const Home: NextPage = () => {
 
       <HeroSection onHandleGetStarted={handleGetStarted} />
       <HomeCarousel />
-      <Footer />
+      <MidSection/>
```
Comment (abdelhakeem) [inline pages/index.tsx]: Everything looks great! Just need to swap these two, the mid section should come before the carousel. Also need to fix the mobile version, should be stacked in a column and full-width on small screens:

small-screen

**Comment by RajGM** [comment]
They do stack up in rows.
But with a screen a lot less width we need to put that in the column.

stack2
stack

**Review by RajGM — COMMENTED** [review]
Updated according to the last requirements.

**Comment by RajGM** [comment]
Hi @abdelhakeem 
You can check now.

**Comment by abdelhakeem** [comment]
Sorry for the misunderstanding, what I meant is that they should be displayed _full-width_ when stacked in a column, like this:

full-width

Also, as long as they're stacked in a  row, the images should be visible.

**Comment by RajGM** [comment]
Hi @abdelhakeem 
I have tried that but if we make them full width then we are having a lot of free space for text and it doesn't look nice.
stack3

**Comment by RajGM** [comment]
Either we need more text or make the fonts bigger, the former looks nice but the latter doesn't look good.

**Comment by abdelhakeem** [comment]
Alright, I think we can merge this, are there any pending changes?

**Comment by RajGM** [comment]
No, there aren't any pending changes.

**Pull request merged**